setup.py

```diff
@@ -9,6 +8,18 @@
 MAINTAINER_EMAIL = 'leah.wasser@colorado.edu'
 VERSION = '0.0.2-git'
 
+requirements = [
+    'download',
+    'geopandas',
+    'matplotlib',
+    'numpy',
+    'pandas',
+    'rasterio',
+    'shapely',
+    'scikit-image',
+    'tqdm'
```

Comment:
we might be able to drop tqdm although the message is pretty nice  when you download. i will think about this.  Also can you kindly update your branch from master as we are on version .3 now!! :)